src/components/EditorWrapper.vue

```diff
@@ -638,7 +638,7 @@ export default {
 			#editor {
 				overflow: auto;
 				// Fix for IE11 issue where the menubar sometimes was positioned under the text
-				z-index: 1000;
+				z-index: 20;
```

Comment:
Mind to drop the comment above since with the removed IE11 support this is no longer required and caused side effects the bug the pr fixes ;)

Reply:
Done